analyzer/src/test/assets/package-curations-not-deserializable.yml

```diff
@@ -0,0 +1,21 @@
+---
+- id: "Maven:org.hamcrest:hamcrest-core:1.3"
+  curations:
+    description: "description"
+    homepage_url: "http://home.page"
+    binary_artifact:
+      url: ""
+      hash: ""
+      hash_algorithm: ""
+    source_artifact:
+      url: ""
+      hash: ""
+      hash_algorithm: ""
+    vcs:
+      type: ""
+      url: "http://url.git"
+      revision: ""
+      path: ""
+- id: "Maven:org.hamcrest:hamcrest-library:1.3"
+curations:
+homepage_url: "http://home.page"
```

Comment:
For the purpose of the test the file could be smaller, an empty file or a single "-" should already be sufficient.

Reply:
Single dash didn't work, but I reduced the size...